src/main/frontend/pipeline-console-view/pipeline-console/main/StageNodeLink.tsx

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+
+export interface StageNodeLinkProps {
+    agent: string;
+}
+
+function getAgentUrl(name: string) {
+    // Wrap built-in in brackets
+    const id = name == "built-in" ? "(built-in)" : name;
+    const rootPath = document.head.dataset.rootUrl
```

Comment:
When I run this locally it needs to be all in lowercase:
```suggestion
    const rootPath = document.head.dataset.rooturl
```